tests/test_abinit.py

```diff
@@ -52,6 +52,14 @@ def test_abinit_het_and_backproject():
         )
     )
 
+    args = analyze.add_args(argparse.ArgumentParser()).parse_args(
```

Comment:
`analyze` added after an `abinit_het` run to emulate the scenario of the user for issue #237